services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/shared_handlers.py

```diff
@@ -10,6 +10,24 @@
 logger = logging.getLogger(__name__)
 
 
+async def cleanup_containers_and_volumes(
+    shared_store: SharedStore, settings: DynamicSidecarSettings
+) -> None:
+    cleanup_command = (
+        "docker-compose --project-name {project} --file {file_path} rm --force -v"
```

Comment:
The `-v` is not impacting the input/output/workspace right?

Reply:
I've also tested it, and I see there are no issues with state/inputs/outputs folders. Even though these anonymous volumes are attached to both containers and dy-sidecar.